### ISSUE TITLE
Cannot generate new table with a simple string column with v0.1.18

### ISSUE THREAD

**Issue by ghost** [binis2/code-generation#4]
```
[WARNING] 
java.lang.NullPointerException
    at net.binis.codegen.enrich.handler.ModifierEnricherHandler.enrich (ModifierEnricherHandler.java:142)
    at net.binis.codegen.generation.core.Helpers.lambda$handleEnrichers$64 (Helpers.java:770)
    at java.util.ArrayList.forEach (ArrayList.java:1541)
    at net.binis.codegen.generation.core.Helpers.handleEnrichers (Helpers.java:770)
    at java.util.stream.ForEachOps$ForEachOp$OfRef.accept (ForEachOps.java:183)
    at java.util.stream.ReferencePipeline$2$1.accept (ReferencePipeline.java:177)
    at java.util.stream.ReferencePipeline$2$1.accept (ReferencePipeline.java:177)
    at java.util.HashMap$ValueSpliterator.forEachRemaining (HashMap.java:1675)
    at java.util.stream.AbstractPipeline.copyInto (AbstractPipeline.java:484)
    at java.util.stream.AbstractPipeline.wrapAndCopyInto (AbstractPipeline.java:474)
    at java.util.stream.ForEachOps$ForEachOp.evaluateSequential (ForEachOps.java:150)
    at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential (ForEachOps.java:173)
    at java.util.stream.AbstractPipeline.evaluate (AbstractPipeline.java:234)
    at java.util.stream.ReferencePipeline.forEach (ReferencePipeline.java:497)
    at net.binis.codegen.CodeGen.processFiles (CodeGen.java:135)
    at net.binis.codegen.CodeGen.main (CodeGen.java:78)
    at org.codehaus.mojo.exec.ExecJavaMojo$1.run (ExecJavaMojo.java:254)
    at java.lang.Thread.run (Thread.java:834)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  7.767 s
[INFO] Finished at: 2022-04-25T15:55:18-04:00
[INFO] ------------------------------------------------------------------------

```

**Comment by vgaber-spiral** [comment]
It seems like I couldn't add any new entity. Tried to generate prototype for the attached  test entity.

**Comment by vgaber-spiral** [comment]
It seems like I couldn't add any new entity. Tried to generate prototype for the following test entity:
```
package com.spiralbank.prototype.test.objects;

import com.spiralbank.core.annotaions.ConsoleEntity;
import com.spiralbank.db.discriminator.IdDiscriminator;
import com.spiralbank.db.listener.PostListener;
import com.spiralbank.prototype.core.base.BaseEntityPrototype;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;
import net.binis.codegen.annotation.CodeConstant;
import net.binis.codegen.annotation.CodePrototype;
import net.binis.codegen.spring.BaseEntityModifier;

import javax.persistence.Entity;
import javax.persistence.EntityListeners;
import javax.persistence.Table;

@CodePrototype(
        classGetters = false,
        classSetters = false,
        interfaceSetters = false,
        implementationPackage = "com.spiralbank.db.entity",
        baseModifierClass = BaseEntityModifier.class,
        basePath = "../modules/core/src/main/java")
@Entity(name = TestEntityPrototype.TABLE_NAME)
@IdDiscriminator(93)
@ConsoleEntity(name = "Test")
@EntityListeners(PostListener.class)
@Table(name = TestEntityPrototype.TABLE_NAME)
public interface TestEntityPrototype extends BaseEntityPrototype {

    @CodeConstant(isPublic = false)
    long serialVersionUID = -8416758612935925062L;

    String TABLE_NAME = "test_entity";

    String testColumn();

    @Data
    @EqualsAndHashCode(callSuper = true, onlyExplicitlyIncluded = true)
    @ToString(onlyExplicitlyIncluded = true)
    abstract class ClassAnnotations extends BaseClassAnnotations {}
}
```

**Comment by binis2** [comment]
```
    **@CodeClassAnnotations**
    @Data
    @EqualsAndHashCode(callSuper = true, onlyExplicitlyIncluded = true)
    @ToString(onlyExplicitlyIncluded = true)
    abstract class ClassAnnotations extends BaseClassAnnotations {
```
Try declaring this way with versions of the lib >= 0.1.18

**Comment by binis2** [comment]
Fixed with 0.1.21. Thanks.
